pytentiostat/main.py

```diff
@@ -3,19 +3,68 @@
 from pytentiostat.reporter import save_data_to_file
 from pytentiostat.tester import experiment
 from pytentiostat.routines import startup_routine, closing_routine
+from pytentiostat.plotter import plot_updater, plot_initializer
+import matplotlib.pyplot as plt
+import sys
+import time
+
+
+class BoardCom:
+    def __init__(self):
+        self.com, self.board, self.pin_a0, self.pin_a2, self.pin_d9 = startup_routine()
+
+    def configure_board(self):
+        self.com, self.board, self.pin_a0, self.pin_a2, self.pin_d9 = startup_routine()
+
 
 config_data = parse_config_file()
 
-# Initialize Experiment
-com, board, a0, a2, d9 = startup_routine()
+try:
+    # Initialize Experiment
+    board_instance = BoardCom()
+    # com, board, a0, a2, d9 = startup_routine()
+except KeyboardInterrupt:
+    sys.exit("Stopped. Make any changes and then restart the software.\nExiting...")
+
+board_objects = (board_instance.pin_a0, board_instance.pin_a2, board_instance.pin_d9)
+
+reconfig = "sting"
+
+while True:
+    try:
+        # Run the experiment and get the config_data
+        times, voltages, currents = experiment(config_data, *board_objects)
+        break
+    except KeyboardInterrupt:
+        reconfig = input("Experiment stopped. To restart the same experiment, press Enter. \n"
+                         "If you want to do a different experiment, type \"New\"\n"
```

Comment:
maybe let them know that the experiment will read hte new config file.  This is not clear.

''to run a new experiment edit the config file and then type new and hit enter" or sthg?

Reply:
I added more details here. I think I can come up with a better way to handle this once we include the pauses in the program but hopefully this will do for now.